src/test/java/lotto/domain/model/LottoTest.java

```diff
@@ -22,43 +21,54 @@ class LottoTest {
     @Test
     public void 인자로들어온_로또번호가_3개_일치한다() {
         Lotto lotto = new Lotto(40, 41, 42, 43, 44, 45);
-        assertThat(lotto.findMatchCount(List.of(35, 36, 37, 43, 44, 45)))
+        assertThat(lotto.findMatchCount(new Lotto(35, 36, 37, 43, 44, 45)))
             .isEqualTo(3);
```

Comment:
```suggestion
        assertThat(lotto.findMatchCount(new Lotto(35, 36, 37, 43, 44, 45)))
.isEqualTo(Rank.FIFTH);
```
숫자가 아니라 Rank로 비교하면 어떨까?